### ISSUE TITLE
kamilio container cannot launch

### ISSUE THREAD

**Issue by dcale** [jamhed/kazoo-docker#4]
the build runs through but when I try to run.sh all containers, kamilio instantly crashes:

```bash
$ sudo docker logs a3dcffaa0a8c
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !MY_HOSTNAME!kamailio.kazoo!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !MY_IP_ADDRESS!172.18.0.4!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !MY_AMQP_URL!kazoo://guest:guest@rabbitmq.kazoo:5672!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !MY_WEBSOCKET_DOMAIN!2600hz.com!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !UDP_SIP!udp:172.18.0.4:5060!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TCP_SIP!tcp:172.18.0.4:5060!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TLS_SIP!tls:172.18.0.4:5061!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !UDP_ALG_SIP!udp:172.18.0.4:7000!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TCP_ALG_SIP!tcp:172.18.0.4:7000!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TLS_ALG_SIP!tls:172.18.0.4:7001!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TCP_WS!tcp:172.18.0.4:5064!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !UDP_WS_SIP!udp:172.18.0.4:5064!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !TLS_WSS!tls:172.18.0.4:5065!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !UDP_WSS_SIP!udp:172.18.0.4:5065!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !KAZOO_DB_URL!text:///etc/kamailio/dbtext!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !MAX_WHILE_LOOPS!500!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !OPENBTS_AUTH_SECRET!b3a54fa8317c7d9cb1d89d8970947b30eda273124d97fc3a079ccc98ecc2569b!g
 0(1) INFO: <core> [core/ppcfg.c:82]: pp_subst_add(): ### added subst expression: !ANTIFLOOD_CACHE_PERIOD!600!g
 0(1) WARNING: <core> [core/cfg.y:3387]: warn_at(): warning in config file /etc/kamailio/default.cfg, line 63, column 13: raw socket support not compiled in
 0(1) ERROR: <core> [core/sr_module.c:571]: load_module(): could not find module <outbound> in </usr/local/lib64/kamailio/modules>
 0(1) : <core> [core/cfg.y:3407]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 90, column 12-24: failed to load module
 0(1) ERROR: <core> [core/sr_module.c:571]: load_module(): could not find module <uuid> in </usr/local/lib64/kamailio/modules>
 0(1) : <core> [core/cfg.y:3407]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 99, column 12-20: failed to load module
 0(1) ERROR: <core> [core/sr_module.c:571]: load_module(): could not find module <kazoo> in </usr/local/lib64/kamailio/modules>
 0(1) : <core> [core/cfg.y:3407]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 166, column 12-21: failed to load module
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 167, column 50: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 168, column 51: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 178, column 43: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 180, column 63: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 181, column 47: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 182, column 46: Can't set module parameter
 0(1) ERROR: <core> [core/modparam.c:152]: set_mod_param_regex(): No module matching <kazoo> found
 0(1) : <core> [core/cfg.y:3410]: yyerror_at(): parse error in config file /etc/kamailio/default.cfg, line 183, column 39: Can't set module parameter
 0(1) ERROR: <core> [core/pvapi.c:828]: pv_parse_spec2(): error searching pvar "uuid"
 0(1) ERROR: <core> [core/pvapi.c:1032]: pv_parse_spec2(): wrong char [g/103] in [$(uuid(g){s.rm,-})] at [7 (2)]
 0(1) : <core> [core/cfg.y:3407]: yyerror_at(): parse error in config file /etc/kamailio/registrar-role.cfg, line 111, column 19-36: Can't get from cache: $(uuid(g){s.rm,-})
ERROR: bad config file (11 errors)
 0(1) WARNING: <core> [core/ppcfg.c:221]: pp_ifdef_level_check(): different number of preprocessor directives: N(#!IF[N]DEF) - N(#!ENDIF) = 1
 0(1) INFO: <core> [core/sctp_core.c:53]: sctp_core_destroy(): SCTP API not initialized
```

**Comment by ghost** [comment]
getting the same error.

**Comment by jamhed** [comment]
It seems config wasn't changed automatically how it should. @onezerooneMod, how do you start the container? There are scripts to build/start it.

**Comment by UzairM** [comment]
I use the provided build scripts and run scripts.

**Comment by dcale** [comment]
Same here... Just followed the readme.md

**Comment by dcale** [comment]
When can we expect a fix/reaction for this issue?

**Comment by jamhed** [comment]
@dscale, @UzairM  this repo was moved to  https://github.com/2600hz/docker

**Comment by jamhed** [comment]
And I've updated this to latest too.

**Comment by jamhed** [comment]
@dcale, @UzairM Also it seems the issue is related with Kamailio version. I've fixed it to 4.4.4 for time being, should work as expected.
